### ISSUE TITLE
Scrubber and a few other elements do not render on android 5.1

### ISSUE THREAD

**Issue by gonemad** [negusoft/holoaccent#40]
I believe this is a similar issue to things not rendering correctly in 5.0.  Looking at the scrubber_control_selector i see it directly uses the *_reference drawables.  This no longer works ion 5.1

**Comment by blurkidi** [comment]
Actually, intercepting drawables on 5.1 doesn't work anymore. But color intercepting does.

The new approach is to use the material themes for android 5.0+, while keeping the benefits of HoloAccent. Please have a look at the 'dev' branch and let me know if it is working correctly.

I have been working on a new library too. It follows the same principles as HoloAccent and it backports the Material themes to android 4.0+. It is called GreenMatter and it is in development stage, but I suggest that you consider replacing HoloAccent by it:

https://github.com/negusoft/GreenMatter

**Comment by gonemad** [comment]
Thats not true.  level-list still can get intercepted (i been using it for intercepting for awhile)

My users want holo so GreenMatter is not an option.  Anyway i just posted this to let you know its broken
